components/automate-cli/cmd/chef-automate/applications.go

```diff
@@ -9,14 +14,27 @@ import (
 	eventgw "github.com/chef/automate/api/config/event_gateway"
 	"github.com/chef/automate/api/config/gateway"
 	w "github.com/chef/automate/api/config/shared/wrappers"
+	"github.com/chef/automate/api/external/applications"
+	"github.com/chef/automate/components/automate-cli/pkg/client/apiclient"
+	"github.com/chef/automate/components/automate-cli/pkg/status"
 	"github.com/chef/automate/components/automate-deployment/pkg/client"
 )
 
+var (
+	thresholdMinutes int64
+	format           string
+)
```

Comment:
[nit] We usually put these in a small struct:

```golang
var opts = struct{
	thresholdMinutes int64
	format           string
}{}
```

Reply:
I can do that change if that's the standard. 🥇